composer.json

```diff
@@ -24,9 +24,6 @@
     "require-dev": {
         "phpunit/phpunit": "^6.4"
     },
-    "require-dev": {
-        "phpunit/phpunit": "^6.4"
-    },
```

Comment:
Changes out of scope should be in separate commits (yeah, we usually squash, but sometimes we don't).